docs/source/index.rst

```diff
@@ -29,7 +29,7 @@ Dask emphasizes the following virtues:
    feedback and diagnostics to aid humans
 
 
-.. image:: images/dask-overview.svg
+.. image:: images/dask-overview.png
```

Comment:
I think we'll want these to be SVG before merging

Reply:
I asked Christiana about this image, she's going to reach out to the designer about getting the SVG version fixed so `Dask Array` is not in a different color. The rest look good, though!